### ISSUE TITLE
How does react-player show thumbnail/preview of video?

### ISSUE THREAD

**Issue by franceandski** [cookpete/react-player#1138]
#### Current Behavior
This is not a bug report, but rather a question of how react-player works. Specifically on Safari in iOS, when loading a simple video using react-player like so: `<ReactPlayer controls url="/api/video.mp4" />`, react-player was able to show a thumbnail/preview of the video without passing in props like light or poster. However, if I were to create a `<video>` html element myself (outside of ReactPlayer) like so: `<video controls src="/api/video.mp4"></video>`, Safari does not render a thumbnail/preview of the video. Inspecting the `<video>` element created by ReactPlayer and copying the same attributes did not make the thumbnail/preview show up either.

Here is a screenshot from Safari, where the first video is rendered with `ReactPlayer` and the second with a html `<video>` tag.
<img width="345" src="https://user-images.githubusercontent.com/77610369/104892199-be397d00-5926-11eb-8b9e-70d2cfbfc556.png">

What is react-player doing behind the scenes to allow the showing of the thumbnail/preview of the video?

#### Environment
- URL attempting to play: https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/360/Big_Buck_Bunny_360_10s_1MB.mp4
- Browser: Safari
- Operating system: iOS

**Comment by cookpete** [comment]
My guess is that it’s probably this?

https://github.com/cookpete/react-player/blob/e23cb29a544a8cab1f7e4ee25349641c3247410d/src/players/FilePlayer.js#L26-L28

More info: https://github.com/cookpete/react-player/issues/340

**Comment by franceandski** [comment]
I added a couple of console.log() statements to the load function in FilePlayer, but it looks like none of the if conditions in the function are true so nothing else gets called. 

With your suggestion to look at #340 I found that Player's handleReady function is called, but the onReady() function it tries to call doesn't seem to be defined and none of the if statements that come after is true.

Any other thoughts?
